tech-far-hub/package.json

```diff
@@ -33,18 +38,25 @@
     "react": "^18.2.0",
     "react-children-utilities": "^2.8.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.7.1",
     "react-use": "^17.4.0",
     "remark-gfm": "^1.0.0",
     "sass": "^1.56.1",
     "ts-deep-pick": "^0.2.2",
     "uswds": "^2.13.3"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
     "@types/node": "^18.11.9",
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.9",
+    "c8": "^7.12.0",
```

Comment:
Had to look this up! Nice find.. would love to know how actionable the coverage reports are